### ISSUE TITLE
Precompile warning on DFGVariable

### ISSUE THREAD

**Issue by dehann** [JuliaRobotics/DistributedFactorGraphs.jl#252]
```julia
WARNING: Method definition (::Type{DistributedFactorGraphs.VariableNodeData{T} where T<:DistributedFactorGraphs.InferenceVariable})
(Array{Float64, 2}, Array{Float64, 2}, Array{Symbol, 1}, Array{Int64, 1}, Int64, Bool, Symbol, 
Array{Symbol, 1}, T<:DistributedFactorGraphs.InferenceVariable, Bool, Float64, Bool, Bool, 
Int64) where {T<:DistributedFactorGraphs.InferenceVariable} 
in module DistributedFactorGraphs at /home/travis/.julia/packages/
DistributedFactorGraphs/G2Wc2/src/entities/DFGVariable.jl:11 overwritten at /home/travis/
.julia/packages/DistributedFactorGraphs/G2Wc2/src/entities/DFGVariable.jl:41.

  ** incremental compilation may be fatally broken for this module **
```

**Comment by GearsAD** [comment]
Ah, yeah - that is the constructor. Don't think it's serious, but will try include a fix for 0.6

**Comment by Affie** [comment]
I haven't seen this, so it might be fixed by #291
